AutomatedTesting/Gem/PythonTests/largeworlds/dyn_veg/EditorScripts/PhysXColliderSurfaceTagEmitter_E2E_Editor.py

```diff
@@ -153,23 +154,29 @@ def run_test(self):
             self.test_success = self.test_success and baseline_success
 
         # Setup collider entity with a PhysX Mesh
-        test_physx_mesh_asset_path = asset.AssetCatalogRequestBus(
+        test_physx_mesh_asset_id = asset.AssetCatalogRequestBus(
             bus.Broadcast, "GetAssetIdByPath", os.path.join("levels", "physics",
                                                             "c4044697_material_perfacematerialvalidation",
                                                             "test.pxmesh"), math.Uuid(), False)
-        hydra.get_set_test(collider_entity, 0, "Shape Configuration|Shape", 7)
-        hydra.get_set_test(collider_entity, 0, "Shape Configuration|Asset|PhysX Mesh", test_physx_mesh_asset_path)
+
+        # Remove/re-add component due to LYN-5496
```

Comment:
Depending on the lifecycle of this, it might be worth making a GHI mirror?

Reply:
Yeah, I'll see what amzn-sean thinks. Logged into JIRA for now so I could link the failing test ticket that has internal URLs for logs from the nightly test job runs.